### DISCUSSION TITLE
Quantidade de fullnodes do Drex

### DISCUSSION THREAD

**Discussion by ghost** [bacen/pilotord-kit-onboarding#66] [Q&A]
Na arquitetura do piloto do Drex está sendo utilizado o consenso QBFT e 6 validadores.

[https://github.com/bacen/pilotord-kit-onboarding/blob/main/arquitetura.md#topologia](url)

E na documentação do Besu na parte do consenso QBFT, diz que é necessario 3/4 dos nodes para o consenso da rede.

[https://besu.hyperledger.org/private-networks/how-to/configure/consensus/qbft#block-time](url)

No lançamento público do Drex, haverá mais validadores para assegurar a rede?

**Comment by aldenio** [comment]
Prezado @sandrogomes055,

Ainda não há uma previsão oficial de lançamento público do Drex. O projeto em execução no momento é um piloto que tem por finalidade testar as tecnologias, desenvolver o conhecimento e prover insumos para decisões futuras sobre o DREX.
